app/server/tests/integration/utils.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import Redis from "ioredis";
+
+const fullUrl = (url: string) => `http://localhost:4000/${url}`;
+const redisUrl = "redis://localhost:6379";
+
+// Do not fail on error status - let individual tests deal with error cases
+const validateStatus = () => true;
+const standardHeaders = { "Content-Type": "application/json" };
+
+export const get = async (url: string) => {
+    return axios.get(fullUrl(url), { headers: standardHeaders, validateStatus });
+};
+
+export const post = (url: string, payload: any, cookie: string) => {
```

Comment:
do you want to add an "allow explicit any" here? Don't we normally have something to allow this in tests generally?

Reply:
I think that should be sorted now